### DISCUSSION TITLE
logout user

### DISCUSSION THREAD

**Discussion by jshen28** [OpenIDC/mod_auth_openidc#645] [Q&A] [answered]
Hello,

  I am able to setup a websso flow and user login is working correctly. But I would like to know how to logout the current user? I see there is logout url: `https://demo.com/auth/realms/myrealm/protocol/openid-connect/logout`, my question is how to configure mod openidc such that it can use this url logout url and logout current session? 

Thank you.

Best,
Norman

**Comment by zandbelt** [comment, marked as answer]
see https://github.com/zmartzone/mod_auth_openidc/wiki#9-how-do-i-logout-users

**Comment by jshen28** [comment]
Thank you , I will try it.

Best,
Norman

On Mon, Aug 16, 2021 at 3:48 PM Hans Zandbelt ***@***.***>
wrote:

> see
> https://github.com/zmartzone/mod_auth_openidc/wiki#9-how-do-i-logout-users
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zmartzone/mod_auth_openidc/discussions/645#discussioncomment-1188598>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADH3XA4TD72YQ4643YW7MRDT5C7DLANCNFSM5CHFX4HQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 

Best Regards,

Jiatong Shen
